### DISCUSSION TITLE
Build Dependancies

### DISCUSSION THREAD

**Discussion by Adubzzy** [transmission/transmission#6912] [General]
Not sure if it was a move from v3 to V4 or V4.0 to v4.05 but when building transmission on FreeBSD it requires around 130 additional packages to be installed for just the build. The package system deletes all the uneeded packages after install but I am curious as to what changed that requires all the additional packages for build. The options i have set are:

transmission-4.0.6             Transmission BitTorrent client
transmission-daemon-4.0.6      Transmission BitTorrent client
transmission-utils-4.0.6       Transmission BitTorrent client
transmission-web-4.0.6         Transmission BitTorrent client

**Comment by andreygursky** [comment]
Just for reference. Recent build of transmission 4.0.6 (complete: daemon/gtk/qt/...) on the official Debian Sid (unstable) amd64 builder (which starts with minimal set of pre-installed packages). transmission build dependencies required:

> 7 upgraded, 443 newly installed, 0 to remove and 10 not upgraded.
Need to get 262 MB of archives.
After this operation, 1166 MB of additional disk space will be used.

https://buildd.debian.org/status/fetch.php?pkg=transmission&arch=amd64&ver=4.0.6%2Bdfsg-1&stamp=1717714426&raw=0
